externs/browser/html5.js

```diff
@@ -5134,28 +5134,28 @@ Plugin.prototype.length;
 Plugin.prototype.name;
 
 /**
- * @see https://html.spec.whatwg.org/multipage/scripting.html#custom-elements
+ * @see https://html.spec.whatwg.org/multipage/custom-elements.html#customelementregistry
  * @constructor
  */
 function CustomElementRegistry() {}
 
 /**
  * @param {string} tagName
- * @param {function(new:HTMLElement)} klass
+ * @param {!function(new:HTMLElement)} klass
```

Comment:
Remove the `!` since `function` is already non-nullable. See https://google.github.io/styleguide/jsguide.html#jsdoc-nullability.

Reply:
Updated.